tla/consensus/ccfraft.tla

```diff
@@ -1322,6 +1455,15 @@ StateTransitionsProp ==
         /\ leadershipState[i] = Leader => leadershipState[i]' \in {Follower, Leader}
         ]_vars
 
+MembershipStateTransitionsProp ==
+    [][\A i \in Servers :
+        \* RetirementCompleted is the terminal state
+        membershipState[i] = RetirementCompleted 
+        => membershipState[i]' = RetirementCompleted]_vars
+        \* Note that all other transitions between retirement phases are permitted
+        \* For instance, a node could go from NotRetiring to RetirementCompleted in one step if it 
```

Comment:
"NotRetiring" = "Active"?

Reply:
Fixed in https://github.com/microsoft/CCF/pull/5919/commits/9659fe35984411f8cdc075ae80c6a12d7a63df79